torchvision/prototype/transforms/_geometry.py

```diff
@@ -744,7 +744,7 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
                 height=params["height"],
                 width=params["width"],
             )
-            if isinstance(inpt, (features.Label, features.SegmentationMask)):
+            if isinstance(inpt, (features.Label, features.OneHotLabel, features.SegmentationMask)):
```

Comment:
Here you make an assumption that seg mask is one-hot encoded. In segmenation task, the mask is not one-hot encoded and thus this may give an unexpected behaviour.